python/cudf/cudf/core/_internals/timezones.py

```diff
@@ -77,99 +76,23 @@ def _find_and_read_tzfile_tzdata(zone_name):
         raise zoneinfo.ZoneInfoNotFoundError(zone_name)
 
 
-def _read_tzfile_as_frame(tzdir, zone_name):
+def _read_tzfile_as_frame(
```

Comment:
*polish*: rename to `_read_tzfile_as_columns`, since we no longer return a frame?

Reply:
Good idea. Change it to that